code/graphics/matrix.cpp

```diff
@@ -16,6 +16,8 @@ matrix4 gr_texture_matrix;
 
 matrix4 gr_env_texture_matrix;
 
+float gr_near_plane;
```

Comment:
Does this need a default initializer?

Reply:
Perhaps not -- according to Lafiel, you cannot render before you set the projection matrix, so it should never be possible to read from it.